### PR TITLE
add 'deprecated' alerts'

### PR DESCRIPTION
**Pull request by ruslandoga**

### CODE REVIEW

**Comment by ruslandoga** [comment]
I'll remove the attempt to allow users on old version to interact, add alerts on all deprecated messages, and localize

**Review by akhrail1996 — COMMENTED** [review]
I just realised I fucked up when releasing minor patch with deprecated alerts support by adding it everywhere except it's needed the most 🙈
fixed in https://github.com/getsince/test5/pull/211/commits/95628270e9afaab1b1ab3ccb92d55119e43c3fa5
releasing it now

**Review by akhrail1996 — CHANGES_REQUESTED** [review]
this works as expected, however I think we need to add `send-contact` here as well

File: priv/gettext/ru/LC_MESSAGES/alerts.po [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+msgstr ""
+"Language: ru\n"
+"Plural-Forms: nplurals=3\n"
+
+#, elixir-autogen, elixir-format
+#: lib/t_web/channels/feed_channel.ex:172
+#: lib/t_web/channels/feed_channel.ex:235
+#: lib/t_web/channels/feed_channel.ex:245
+msgid "Calls are no longer supported, please upgrade."
+msgstr "Звонки больше не работают, пожалуйста обновитесь."
```
Comment (ruslandoga) [inline priv/gettext/ru/LC_MESSAGES/alerts.po:19]: В остальных сообщениях используется "ты" или "вы"?

Comment (akhrail1996) [reply priv/gettext/ru/LC_MESSAGES/alerts.po:19]: везде используется "ты"

Comment (akhrail1996) [reply priv/gettext/ru/LC_MESSAGES/alerts.po:19]: я исправлю

File: lib/t/accounts.ex [hunk 2]
```diff
@@ -444,6 +444,20 @@ defmodule T.Accounts do
     get_user_by_session_token_and_update_version(token, nil, context)
   end
 
+  @spec list_app_versions(Ecto.Bigflake.UUID.t()) :: [String.t()]
+  def list_app_versions(user_id) do
+    UserToken |> where(user_id: ^user_id) |> select([t], t.version) |> Repo.all()
+  end
+
+  @spec has_previous_version?(Ecto.Bigflake.UUID.t()) :: bool
+  def has_previous_version?(user_id) do
```
Comment (akhrail1996) [inline lib/t/accounts.ex:453]: @ruslandoga is this and the command above used anywhere?
couldn't find. can I remove them?

Comment (ruslandoga) [reply lib/t/accounts.ex:453]: Not used.

I wanted to use it for stats, but now I see that we can just read it from DB. I'd be very thankful if you remove them. 

Thank you.

File: priv/gettext/ru/LC_MESSAGES/alerts.po (hunk 1 above)
Comment (ruslandoga) [reply priv/gettext/ru/LC_MESSAGES/alerts.po:19]: спасибо

File: lib/t_web/channels/feed_channel.ex [hunk 3]
```diff
@@ -266,7 +266,7 @@ defmodule TWeb.FeedChannel do
         dgettext("alerts", "Sending contacts is no longer supported, please upgrade.")
       )
 
-    {:reply, {:error, %{alerts: alert}}, socket}
+    {:reply, {:error, %{alert: alert}}, socket}
```
Comment (ruslandoga) [inline lib/t_web/channels/feed_channel.ex:266]: 👍

**Review by akhrail1996 — APPROVED** [review]

**Comment by akhrail1996** [comment]
will merge & deploy upon release

**Pull request merged**